### ISSUE TITLE
Docker Build and runtime

### ISSUE THREAD

**Issue by edperry** [jmhale/okta-awscli#131]
**Is your feature request related to a problem? Please describe.**
When testing python modules, I like to do it as a docker container. Also for application automation and giving the command to other non-technical users is easier to do though docker. Over providing the instructions to install on the various Linux/Windows/Kubernetes servers we run

**Describe the solution you'd like**
Solution provided:  A docker file with instructions on how to use it.

**Describe alternatives you've considered**
Following the standard build dock or python install to user space

**Additional context**
Provided Pull request with the proposed feature request
#130

**Comment by edperry** [comment]
## Run from docker container
This process is taken from gimme-aws-creds and adapted

### Build the image 
```
docker build -t okta-awscli .

```
### Run the image with the command

```
docker run -it --rm -v ~/.aws/credentials:/root/.aws/credentials -v ~/.okta-aws:/root/.okta-aws --profile default okta-awscli iam list-users
```

### if you want to type less you can create an alias

```
alias okta-awscli='docker run -it --rm -v ~/.aws:/root/.aws -v ~/.okta-aws:/root/.okta-aws okta-awscli'
```

and just type 
```
okta-awscli
```

you can add this to you .bashrc 
```
source <PATH TO GIT REPO>/set-alias.bash
```

**Comment by jmhale** [comment]
Closed with #130. Thanks!
